### ISSUE TITLE
Connecting with Owncloud/Nextcloud music plugin (Subsonic API)

### ISSUE THREAD

**Issue by lorenzo93** [SubmarinerApp/Submariner#164]
Hi,

I've just installed a private instance of Nextcloud and via the [music plugin](https://github.com/owncloud/music) is supports Subsonic API. I've connected successfully with different clients (Android, Windows) but I'm having problems using yours.
The interface just says "Could not connect to the server.".
Is there a way to enable a more verbose log?
I've tried to open Submariner from the terminal, hoping that it logs directly on stdout/stderr but is not writing any line.

**Versions:**
 - macOS 13.5.2, MacBook Pro M2 (Apple Silicon)
 - Submariner Version: 2.4.2
 - Download Source: brew

**Comment by NattyNarwhal** [comment]
It should accessible from Console.app. To filter messages from Submariner specifically, you want a search like `process:Submariner subsystem:fr.read-write.Submariner`, make sure to include debug/info messages (shown below), and start capturing:

<img width="453" alt="Screenshot 2023-09-10 at 5 22 11 PM" src="https://github.com/SubmarinerApp/Submariner/assets/3161292/a4257fc4-d59e-423e-b95a-11e4978ce4a4">

**Comment by lorenzo93** [comment]
Hi,
thanks for the info! Didn't even know the existence of the Console app (my first mac).
I've found some log lines that are referring to "connection refused", I'll check the server logs to understand better.
Which kind of authentication is your app using by default?

**Comment by NattyNarwhal** [comment]
By default, token authentication is used by default. OC Music doesn't support it, so make sure that's unchecked when changing the server settings as shown below:

![Screenshot 2023-09-10 at 8 20 32 PM](https://github.com/SubmarinerApp/Submariner/assets/3161292/2802f337-dc5f-403a-9e4f-4097cd81c39a)

**Comment by lorenzo93** [comment]
I'm sorry, I've fixed the problem and it was on my side. I made some mistakes with the reverse proxy configuration.

Just one little point, submarine views some license information that are not correct, since the plugin is free there should be no license requirements.

In the below screenshot, I've changed the domain name due to privacy concerns :)

<img width="418" alt="image" src="https://github.com/SubmarinerApp/Submariner/assets/12275050/8a284005-cf8b-41af-9ec6-1f98df79eb0d">
